app/src/main/java/org/xbmc/kore/ui/BaseActivity.java

```diff
@@ -15,33 +15,50 @@
  */
```

Comment:
Like i said, i think this activity shouldn't be touched, as it doesn't really need to be changed.